sklearn/utils/tests/test_estimator_html_repr.py

```diff
@@ -608,3 +608,20 @@ def test_function_transformer_show_caption(func, expected_name):
     )
     re_compiled = re.compile(p)
     assert re_compiled.search(html_output)
+
```

Comment:
Let's move this file that is currently `sklearn/utils/tests/test_estimator_html_repr.py` into `sklearn/utils/_repr_html/tests/test_estimator.py`.